builds/e2e/e2e.yaml

```diff
@@ -42,9 +42,31 @@ jobs:
     - template: templates/e2e-clear-docker-cached-images.yaml
     - template: templates/e2e-run.yaml
 
-################################################################################	
+################################################################################
```

Comment:
Since we don't have a 64-bit arm agent ready (yet) to run tests, let's remove this job from `e2e.yaml`.

Reply:
Removed